packages/bpk-docs/src/pages/ButtonsPage/ButtonsPage.js

```diff
@@ -49,35 +51,29 @@ const AlignedBpkLargeHelpIcon = alignToLargeButton(TestBpkLargeHelpIcon);
 const AlignedBpkSmallSearchIcon = alignToButton(TestBpkSmallSearchIcon);
 const AlignedBpkLargeSearchIcon = alignToLargeButton(TestBpkLargeSearchIcon);
 
-class LoadingButtonContainer extends Component {
+class LoadingButtonContainer extends React.Component<{}, { loading: boolean }> {
   constructor() {
     super();
 
     this.state = {
       loading: false,
     };
-
-    this.onClick = this.onClick.bind(this);
   }
 
-  onClick() {
+  onClick = () => {
```

Comment:
This is what adding `transform-class-properties` to `.babelrc` is for. There is a chance that it could break consumers builds, but as mentioned above the majority of consumers use `backpack-react-scripts`.